src/directory/directory.mjs

```diff
@@ -1229,27 +1234,37 @@ export const directory = {
           {
             title: 'Social sign-in (OAuth)',
             route: '/lib-v1/auth/social',
-            filters: ['android', 'ios', 'flutter', 'js']
+            filters: ['android', 'ios', 'flutter']
```

Comment:
Why is js removed here?

Reply:
we are changing it to another path for js and rn